x-pack/solutions/search/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_actions/actions/delete/delete_action.tsx

```diff
@@ -5,22 +5,23 @@
  * 2.0.
  */
 
-import { EuiButtonIcon } from '@elastic/eui';
-import { i18n } from '@kbn/i18n';
-import React, { useState } from 'react';
-import { isEndpointPreconfigured } from '../../../../../../utils/preconfigured_endpoint_helper';
+import React from 'react';
 import { useDeleteEndpoint } from '../../../../../../hooks/use_delete_endpoint';
 import { InferenceEndpointUI } from '../../../../types';
 import { ConfirmDeleteEndpointModal } from './confirm_delete_endpoint';
 
 interface DeleteActionProps {
   selectedEndpoint: InferenceEndpointUI;
+  onCancel: () => void;
+  displayModal: boolean;
```

Comment:
nitpick - non-blocking: for some reason it doesn't feel right to get the visibility of this modal from outside.